### DISCUSSION TITLE
Welcome to The-Complete-FAANG-Preparation Discussions!

### DISCUSSION THREAD

**Discussion by AkashSingh3031** [AkashSingh3031/The-Complete-FAANG-Preparation#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AkashSingh3031** [comment]
___________________
𝕋𝕣𝕖𝕖 𝕠𝕗 𝕀𝕟𝕕𝕖𝕩
───────────
```js
.
├── ...
├── 𝑴𝒊𝒔𝒄𝒆𝒍𝒍𝒂𝒏𝒆𝒐𝒖𝒔 𝑺𝒕𝒖𝒇𝒇
|   ├── Aptitude & Reasoning              
|   |   └── ...
|   |
|   ├── Basic Programming MCQs
|   |   └── ...
|   |
|   ├── Puzzles
|   |   └── ...
|   |
|   └── ...
|
|
└── 𝑫𝑺𝑨
|   ├── Data Structures              
|   |   └── ...
|   |
|   ├── Algorithms
|   |   └── ...
|   |
|   ├── 450 DSA by @Love Babbar
|   |   └── ...
|   |
|   ├── FAANG Interview Questions
|   |   ├── Facebook
|   |   ├── Amazon
|   |   ├── Apple
|   |   ├── Netflix
|   |   ├── Google
|   |   ├── Others
|   |   └── ...
|   |
|   └── ...
|
|
└── 𝑪𝒐𝒎𝒑𝒆𝒕𝒊𝒕𝒊𝒗𝒆 𝑷𝒓𝒐𝒈𝒓𝒂𝒎𝒎𝒊𝒏𝒈
|   ├── HackerRank              
|   |   └── ...
|   |
|   ├── GeeksforGeeks
|   |   └── ...
|   |
|   ├── HackerEarth
|   |   └── ...
|   |
|   ├── CodeChef              
|   |   └── ...
|   |
|   ├── CodeForce
|   |   └── ...
|   |
|   ├── LeetCode
|   |   └── ...
|   |
|   ├── InterviewBit
|   |   └── ...
|   |
|   └── ...
|
|
└── 𝑻𝒆𝒄𝒉𝒏𝒊𝒄𝒂𝒍 𝑺𝒖𝒃𝒋𝒆𝒄𝒕
|   ├── Object Oriented Programming (OOPs)              
|   |   └── ...
|   |
|   ├── Operating System (OS)
|   |   └── ...
|   |
|   ├── Database Management System (DBMS)
|   |   └── ...
|   |
|   ├── Structured Query Language (SQL)              
|   |   └── ...
|   |
|   ├── Computer Network (CN)
|   |   └── ...
|   |
|   ├── Interview Questions
|   |   └── ...
|   |
|   └── ...
|
|
└── 𝑳𝒐𝒘 𝑳𝒆𝒗𝒆𝒍 𝑫𝒆𝒔𝒊𝒈𝒏   
│   └── ...
|
|
└── 𝑷𝒓𝒐𝒋𝒆𝒄𝒕
|   ├── Machine Learning & Data Science              
|   |   └── ...
|   |
|   ├── Internet of Things (IoT)
|   |   └── ...
|   |
|   ├── Web Development
|   |   └── ...
|   |
|   ├── Mobile Development              
|   |   └── ...
|   |
|   └── ...
└── ...

```

**Reply by buildwithmalik** [reply]
The content looks empty, do the admins have a plan of adding content or does that depend on contributors?

**Reply by AkashSingh3031** [reply]
@AbdulMalikDev 
I also add resources and all folders filled only possible if everyone contributes, because everyone doesn't know everything.

**Reply by buildwithmalik** [reply]
ok I'll add 450 dsa problems i have some in python

**Reply by AkashSingh3031** [reply]
Ok 👍

**Reply by GJuceviciute** [reply]
Hey! I'm new to the open source contributing thing, but I've solved quite a few leetcode problems, so maybe I could help with this project? If I understand it correctly, I just need to copy paste my code, name the files with leetcode questions and put them in the right folders? How does numbering the problems work? Is it just the order problems were contributed in? I noticed the same problem (two sum) having 0003 at the start of the file name in Java folder, but 0001 in Python. Thanks

**Reply by AkashSingh3031** [reply]
> Hey! I'm new to the open source contributing thing, but I've solved quite a few leetcode problems, so maybe I could help with this project? If I understand it correctly, I just need to copy paste my code, name the files with leetcode questions and put them in the right folders? How does numbering the problems work? Is it just the order problems were contributed in? I noticed the same problem (two sum) having 0003 at the start of the file name in Java folder, but 0001 in Python. Thanks

@GJuceviciute 
Yes, you solve the question on `LeetCode` and Running & Accepted code Send code here with the correct number in LeetCode
And thanks for mention `wrong numbering in Java`.
I will correct this

**Reply by VishnuSastryHK** [reply]
Hey! I would like to add some solutions in C++ to Love Bhaiya's 450 DSA. Also, I would like to contribute to the Data Structures part.

**Reply by AkashSingh3031** [reply]
> Hey! I would like to add some solutions in C++ to Love Bhaiya's 450 DSA. Also, I would like to contribute to the Data Structures part.

You are welcome...
